pylint_odoo/misc.py

```diff
@@ -112,14 +113,27 @@ def get_manifest_file(self, node_file):
 
     def set_ext_files(self):
         """Create `self.ext_files` dictionary with {extension_file: [files]}
+            and exclude files using --ignore and --ignore-patterns parameters
         """
         self.ext_files = {}
         for root, _, filenames in os.walk(self.module_path, followlinks=True):
             for filename in filenames:
+                find = True
                 fext = os.path.splitext(filename)[1].lower()
                 fname_rel = os.path.relpath(
                     os.path.join(root, filename), self.module_path)
-                self.ext_files.setdefault(fext, []).append(fname_rel)
+                name = os.path.join(self.module_path, fname_rel)
```

Comment:
Output of pdb

```
(Pdb++) os.path.join(root, filename)
'/tmp/test_repo/test_module/res_users.xml'
(Pdb++) name
'/tmp/test_repo/test_module/res_users.xml'
```
They are the same values!
I can remove the variable `name` and use `os.path.join (root, filename)` directly

Is that what you mean?